apps/wallet/tests/staking.spec.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect, test } from './fixtures';
+import { createWallet } from './utils/auth';
+
+const TEST_TIMEOUT = 65 * 1000;
+const STAKE_AMOUNT = 100;
+
+test('staking', async ({ page, extensionUrl }) => {
+    test.setTimeout(TEST_TIMEOUT);
+
+    await createWallet(page, extensionUrl);
+
+    await page.getByTestId('faucet-request-button').click();
+    await expect(page.getByText(/SUI Received/i)).toBeVisible();
```

Comment:
These are testing the faucet button, not staking. I'd remove these assertions. Most ideally I'd actually run the faucet in the test instead of through the UI but that's not essential.